images/Dockerfile.base

```diff
@@ -29,6 +29,7 @@ RUN pip install docker-compose==1.6
 
 # Install golem binary
 COPY golem /usr/local/bin/golem_runner
+COPY golem /usr/local/bin/golem_tapper
```

Comment:
I had that thought, let me try it out and make sure it doesn't cause any problems.